app/templates/index.html

```diff
@@ -92,33 +92,21 @@ <h2 class="marketplace-homepage-subheading">Find technology or people for digita
   </div>
     <div class="supplier-messages column-one-third">
       <aside role="complementary" aria-labelledby="supplier-message-heading">
-        
+
       {% if temporary_message or current_user.role == 'supplier' or dos_is_live %}
           <h2 id="supplier-message-heading">Sell services</h2>
       {% endif %}
-        
-      {% if temporary_message %}
-        {%
-          with
-            heading = temporary_message.heading,
-            subheading = temporary_message.subheading,
-            messages = temporary_message.messages,
-            message = temporary_message.message
-        %}
-          {% include "toolkit/{}.html".format(temporary_message.template_name) %}
-        {% endwith %}
-      {% endif %}
-        
+
       {% if dos_is_live %}
+
         <div class="padding-bottom-small">
           <p>
             <a href="/digital-outcomes-and-specialists/opportunities" class="top-level-link">
-              Digital Outcomes and Specialists opportunities
+              View Digital Outcomes and Specialists opportunities
             </a>
           </p>
         </div>
-        
-        {% if current_user.role != 'supplier' %}
+      {% set selling %}
```

Comment:
I'm not completely sure what you mean, unless you are talking about the statement below this one which is gone.